src/devices/Bmx280/BmxBase.cs

```diff
@@ -6,44 +6,37 @@
 //Formulas and code examples can also be found in the datasheet http://www.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf
 
 using System;
+using System.Buffers.Binary;
 using System.Device.I2c;
 using System.Threading.Tasks;
-using System.Buffers.Binary;
 
-namespace Iot.Device.Bmp280
+namespace Iot.Device.Bmx280
```

Comment:
I'm wondering if X should be capital - otherwise Bmx sounds like it is part of the name 😄 (no strong opinion either way though)

Reply:
Everywhere else we are using lower case x to replace characters. So I went for the lower case one

Reply:
fair enough

Reply:
Interesting point though, We can log an issue and change this to capital X in the places where we do this as I agree with @krwq that it would be more clear that we are talking about a base class instead of a individual binding.

Reply:
yes, the previous case happened to end with number just before the `x` so it didn't matter. fine to keep as is for this PR - let's observe this closer once we get more devices